examples/minwebgl/outline/src/main.rs

```diff
@@ -551,9 +622,17 @@ async fn run() -> Result< (), gl::WebglError >
   let window = gl::web_sys::window().unwrap();
   let document = window.document().unwrap();
 
-  let gltf_path = "bike.glb";
+  let gltf_path = "2017_porsche_911_turbo_s_exclusive_series_991.2.glb";
   let gltf = load( &document, gltf_path, &renderer.gl ).await?;
   let scenes = gltf.scenes.clone();
+  for node in &scenes[ 0 ].borrow().children
+  {
+    node.borrow_mut().set_center_to_origin();
+    node.borrow_mut().normalize_scale();
```

Comment:
Remove it. Both of them directly change the world matrix and it should not be allowed. They should affect only local matrix. I'll change it in this [PR](https://github.com/Wandalen/cgtools/pull/153)